sdk/servicebus/service-bus/test/internal/backupMessageSettlement.spec.ts

```diff
@@ -310,22 +329,30 @@ describe("Message settlement After Receiver is Closed - Through ManagementLink",
         "MessageId is different than expected"
       );
 
+      testLogger.info(`Attempting to complete the message: ${deadLetterMsgsBatch[0].messageId}`);
       await receiver.completeMessage(deadLetterMsgsBatch[0]);
-
       await testPeekMsgsLength(deadLetterReceiver, 0);
     } else {
       const messageBatch = await receiver.receiveMessages(1);
-      await receiver.completeMessage(messageBatch[0]);
 
+      testLogger.info(`Attempting to complete the message: ${messageBatch[0].messageId}`);
+      await receiver.completeMessage(messageBatch[0]);
       await testPeekMsgsLength(receiver, 0);
     }
+
+    testLogger.info(`Done testing dead letter`);
   }
 
-  it(
+  it.only(
```

Comment:
.

Reply:
Oh...no. Maybe I'm not running the tests I want then. Will fix!

Reply:
Okay good, I still had it over the critical tests. .only's removed in the next push so the next test run will be against the whole suite (which could be causing issues).

Reply:
Removed